zad3/app/helpers/bestiary_helper.rb

```diff
@@ -1,4 +1,12 @@
 # frozen_string_literal: true
 
 module BestiaryHelper
+    def table_header
+        '<tr>
+            <th>ID</th>
+            <th>Name</th>
+            <th>Location</th>
+            <th>Best Slayer</th>
+        </tr>'.html_safe
```

Comment:
Miejsce na HTML jest tylko w widoku 😉 W takim helperze możesz sobie zwrócić np `['ID', 'Name', 'Location', 'Best Slayer']`, a dopiero widoku włożyć te dane w HTML

Reply:
Ach zgadza się, początkowo dodałem to w partialu, aczkolwiek potem stwierdziłem że użyje sobie treningowo helpera, ale rzeczywiście mogłem sensowniej skorzystać z helpera :D